tests/test_recreation.py

```diff
@@ -1248,7 +1248,9 @@ def test_bad_predictor_table_header(self):
             file.write('foo,bar,baz\n')
             file.write('a,b,c\n')
 
-        expected_message = "Fields are missing from this table: ['ID', 'PATH', 'TYPE']"
+        expected_message = [(
+            ['predictor_table_path'],
+            'id matched 0 headers, expected at least one')]

```

Comment:
Same comment here about phrasing of the validation message.

Reply:
see https://github.com/natcap/invest/pull/526#discussion_r652766326